lib/pf/web.pm

```diff
@@ -71,7 +72,15 @@ Warning: The list of subroutine is incomplete
 sub i18n {
     my $msgid = shift;
 
-    return gettext($msgid);
+    if(ref($msgid) eq "ARRAY"){
+        my @infos = @$msgid;
+        my $id = shift @infos;
```

Comment:
Unnecessary work just pass the array to i18n_format.

return i18n_format(@$msgid);